src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -120,7 +120,7 @@ struct CartesianCoords : db::ComputeItemTag {
       typelist<Strahlkorper<Frame>, Radius<Frame>, Rhat<Frame>>;
 };
 
-/// dx_radius(i) is \f$\partial r/\partial x^i\f$.
+/// `DxRadius`(i) is \f$\partial r/\partial x^i\f$.
```

Comment:
Change class name to be consistent with derivative function names in scalar wave analytic solution